src/startupscriptgenerator/dotnetcore/main.go

```diff
@@ -48,6 +55,30 @@ func main() {
 		fullDefaultAppFilePath = common.GetValidatedFullPath(*defaultAppFilePathPtr)
 	}
 
+	if (*copyOutputToDifferentDirAndRunPtr == "true") {
```

Comment:
Idea here was to make code changes as less intrusive as possible, so once the copy is done and the paths have been modified appropriately, rest of the existing code should work as it was before